src/test/java/openfasttrack/report/TestPlainTextReport.java

```diff
@@ -158,7 +142,27 @@ private void prepareFailedItemDetails()
         when(itemBMock.getDescription()).thenReturn("desc B1");
         when(itemCMock.getDescription()).thenReturn("desc C1\ndesc C2");
         when(itemDMock.getDescription()).thenReturn("desc D1");
+        when(itemAMock.isDefect()).thenReturn(true);
+        when(itemBMock.isDefect()).thenReturn(true);
+        when(itemCMock.isDefect()).thenReturn(true);
+        when(itemDMock.isDefect()).thenReturn(true);
         when(this.traceMock.getUncoveredItems())
                 .thenReturn(Arrays.asList(itemAMock, itemBMock, itemCMock, itemDMock));
     }
+
+    private String expectFailureDetails()
+    {
+        final StringBuilder builder = new StringBuilder();
+        builder.append("not ok - dsn~bar~1\n") //
+                .append("# desc B1\n") //
+                .append("not ok - req~foo~1\n") //
+                .append("# desc A1\n# desc A2\n# desc A3\n") //
+                .append("not ok - req~zoo~1\n") //
+                .append("# desc D1\n") //
+                .append("not ok - req~zoo~2\n") //
+                .append("# desc C1\n# desc C2\n") //
+                .append("\nnot ok - 6 total, 4 not covered\n");
+
+        return builder.toString();
```

Comment:
`StringBuilder` is not necessary here, simple string concatenation would be easier to read. The Java compiler will use StringBuilder anyway if this would improve performance.